modules/mongodb/src/main/java/org/testcontainers/containers/MongoDBContainer.java

```diff
@@ -17,9 +18,9 @@
 @Slf4j
 public class MongoDBContainer extends GenericContainer<MongoDBContainer> {
 
-    private static final DockerImageName DEFAULT_IMAGE_NAME = DockerImageName.parse("mongo");
+    static final DockerImageName DEFAULT_IMAGE_NAME = DockerImageName.parse("mongo");
 
-    private static final String DEFAULT_TAG = "4.0.10";
+    static final String DEFAULT_TAG = "6.0.4";
```

Comment:
Let's not update the default version due to backward compatibility. If `new MongoDBContainer()` is used then the default version is used. Nowadays, is highly recommend to use `new MongoDBContainer("mongo:6")` instead.

Reply:
I'm not terribly tied to this.  I won't presume to make product level decisions in my first PR.  I can change to "6" if you'd like or revert it to the 4.x version.  But 4.x is *really* old at this point so making it the default seems ... unproductive.  Your call, though.

Reply:
@evanchooly can you revert the default tag to `4.0.10`, please? We know is very old and it belongs to the old-way to do it in Testcontainers. However, we are promoting a lot using `new MongoDBContainer("mongo:6")` :)